src/base/preferences.cpp

```diff
@@ -463,6 +463,16 @@ void Preferences::setServerDomains(const QString &str)
     setValue("Preferences/WebUI/ServerDomains", str);
 }
 
+QString Preferences::getWebUiAddress() const
+{
+    return value("Preferences/WebUI/Address", "*").toString().trimmed();
```

Comment:
Don't understand the logic behind silently trimming in both directions

Reply:
I guess its to silently fix whitespace misktakes made by users either in the config file or in the UI. This was suggested by an earlier reviewer. I'm fine either way, I just want qbit to work in my FreeBSD jail!

Reply:
>fix whitespace misktakes made by users either in the config file or in the UI.

Yes, it is.